### DISCUSSION TITLE
version does't provide valid work with -c and rate

### DISCUSSION THREAD

**Discussion by hexsurfer** [projectdiscovery/naabu#1093] [General]
I've tested, command, -c 50 shows the following:
ps -eLf | grep naabu | grep -v "auto" shows 28 process not 50

**Comment by GeorginaReeder** [comment]
Thanks for this suggestion @hexsurfer , we appreciate it!

**Comment by Mzack9999** [comment]
@hexsurfer a process thread might contain and handle multiple go-routines, this is handled internally by the go scheduler
